exercises/clock/clock_test.rb

```diff
@@ -3,146 +3,300 @@
+  def test_hour_and_minutes_roll_over_continuously
     skip
-    clock = Clock.at(10) - 90
-    assert_equal '08:30', clock.to_s
+    assert '11:01', Clock.at(201, 3001)
   end
 
-  def test_hour_rolls_over_continuously
+  def test_hour_and_minutes_roll_over_to_exactly_midnight
     skip
-    clock = Clock.at(0) + hours(100)
-    assert_equal '04:00', clock.to_s
+    assert '00:00', Clock.at(72, 8640)
   end
 
-  def test_minutes_roll_over_continuously
+  def test_negative_hour
     skip
-    clock = Clock.at(0) + minutes(1723)
-    assert_equal '04:43', clock.to_s
+    assert '23:15', Clock.at(-1, 15)
   end
 
-  def test_hour_and_minutes_roll_over_continuously
+  def test_negative_hour_rolls_over
     skip
-    clock = Clock.at(0) + hours(201) + minutes(3001)
-    assert_equal '11:01', clock.to_s
+    assert '23:00', Clock.at(-25, 0)
   end
 
-  def test_negative_hours_roll_over_continuously
+  def test_negative_hour_rolls_over_continuously
     skip
-    clock = Clock.at(0) - hours(91)
-    assert_equal '05:00', clock.to_s
+    assert '05:00', Clock.at(-91, 0)
+  end
+
+  def test_negative_minutes
+    skip
+    assert '00:20', Clock.at(1, -40)
+  end
+
+  def test_negative_minutes_roll_over
+    skip
+    assert '22:20', Clock.at(1, -160)
   end
 
   def test_negative_minutes_roll_over_continuously
     skip
-    clock = Clock.at(0) - minutes(4820)
-    assert_equal '15:40', clock.to_s
+    assert '16:40', Clock.at(1, -4820)
+  end
+
+  def test_negative_hour_and_minutes_both_roll_over
+    skip
+    assert '20:20', Clock.at(-25, -160)
   end
 
-  def test_negative_hours_and_minutes_both_roll_over_continuously
+  def test_negative_hour_and_minutes_both_roll_over_continuously
     skip
-    clock = Clock.at(0) - hours(121) - minutes(5810)
-    assert_equal '22:10', clock.to_s
+    assert '22:10', Clock.at(-121, -5810)
+  end
+
+  def test_add_minutes
+    skip
+    assert '10:03', Clock.at(10, 0) + 3
   end
 
   def test_add_no_minutes
     skip
-    clock = Clock.at(0) + minutes(0)
-    assert_equal '00:00', clock.to_s
+    assert '06:41', Clock.at(6, 41) + 0
+  end
+
+  def test_add_to_next_hour
+    skip
+    assert '01:25', Clock.at(0, 45) + 40
+  end
+
+  def test_add_more_than_one_hour
+    skip
+    assert '11:01', Clock.at(10, 0) + 61
   end
 
   def test_add_more_than_two_hours_with_carry
     skip
-    clock = Clock.at(0, 45) + minutes(160)
-    assert_equal '03:25', clock.to_s
+    assert '03:25', Clock.at(0, 45) + 160
+  end
+
+  def test_add_across_midnight
+    skip
+    assert '00:01', Clock.at(23, 59) + 2
+  end
+
+  def test_add_more_than_one_day__1500_min_is_equal_to_25_hrs_
+    skip
+    assert '06:32', Clock.at(5, 32) + 1500
   end
 
   def test_add_more_than_two_days
     skip
-    clock = Clock.at(1, 1) + days(2) + hours(10) + minutes(20)
-    assert_equal '11:21', clock.to_s
+    assert '11:21', Clock.at(1, 1) + 3500
+  end
+
+  def test_subtract_minutes
+    skip
+    assert '10:00', Clock.at(10, 3) + -3
```

Comment:
After checking on `x-common`, I retract my comment and think this is fine and within the spirit of the test.